napari/layers/_points_layer/view/properties.py

```diff
@@ -133,10 +127,6 @@ def _on_symbol_change(self, event):
     def _on_size_change(self, event):
         with self.layer.events.size.blocker():
             value = self.layer.size
-            if isinstance(value, Iterable):
-                if isinstance(value, list):
-                    value = np.asarray(value)
-                value = value.mean()
```

Comment:
Just for future reference, for this pattern imho it's better to use `value = np.mean(value)`, which will work equally well with lists and arrays.